examples/with-css-modules/tsconfig.json

```diff
@@ -7,6 +7,6 @@
     "moduleResolution": "bundler",
     "isolatedModules": true,
     "noEmit": true,
-    "jsx": "react-jsx"
+    "jsx": "react"
```

Comment:
We don't want to use the automatic runtime in userland examples?

Reply:
Personally I would like to use the automatic runtime. This is coming from the `docusaurus` branch. Unfortunately Docusaurus still uses React 17. As dealing with multiple React versions in one mono repo is very cumbersome, I’m going to have to downgrade React everywhere. In React 17 the automatic runtime is broken.

The reason this change is applied in this PR already, is because moving the React ESLint configuration up triggered [`react/jsx-uses-react`](https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/jsx-uses-react.md).

Reply:
Should we deprecate 16/17 support in `@markprompt/react` instead?